build/source/driver/multi_driver.f90

```diff
@@ -128,70 +143,78 @@ program multi_driver
-real(dp),parameter        :: safety = 0.01_dp               ! safety factor to ensure logistic function is less than 1
-real(dp),parameter        :: fSmall = epsilon(xMatch)       ! smallest possible value to test
-real(dp),allocatable      :: upArea(:)                      ! area upslope of each HRU
-! general local variables
-real(dp)                  :: fracHRU                        ! fractional area of a given HRU (-)
-integer(i4b)              :: fileUnit                       ! file unit (output from file_open; a unit not currently used)
-character(LEN=256),allocatable :: dataLines(:)    ! vector of character strings from non-comment lines
-character(LEN=256),allocatable :: chardata(:)     ! vector of character data
-integer(i4b)              :: iWord                          ! loop through words in a string
-logical(lgt)              :: flux_mask(size(flux_meta))     ! mask defining desired flux variables
-real(dp),allocatable      :: zSoilReverseSign(:)            ! height at bottom of each soil layer, negative downwards (m)
-real(dp),dimension(12)    :: greenVegFrac_monthly           ! fraction of green vegetation in each month (0-1)
-real(dp),parameter        :: doubleMissing=-9999._dp        ! missing value
-logical(lgt),parameter    :: overwriteRSMIN=.false.         ! flag to overwrite RSMIN
-real(dp)                  :: notUsed_canopyDepth            ! NOT USED: canopy depth (m)
-real(dp)                  :: notUsed_exposedVAI             ! NOT USED: exposed vegetation area index (m2 m-2)
+real(dp),parameter               :: supersatScale=0.001_dp     ! scaling factor for the logistic function (-)
+real(dp),parameter               :: xMatch = 0.99999_dp        ! point where x-value and function value match (-)
+real(dp),parameter               :: safety = 0.01_dp           ! safety factor to ensure logistic function is less than 1
+real(dp),parameter               :: fSmall = epsilon(xMatch)   ! smallest possible value to test
+! general local variables        
+real(dp)                         :: fracHRU                    ! fractional area of a given HRU (-)
+integer(i4b)                     :: fileUnit                   ! file unit (output from file_open; a unit not currently used)
+character(LEN=256),allocatable   :: dataLines(:)               ! vector of character strings from non-comment lines
+character(LEN=256),allocatable   :: chardata(:)                ! vector of character data
+integer(i4b)                     :: iWord                      ! loop through words in a string
+logical(lgt)                     :: flux_mask(size(flux_meta)) ! mask defining desired flux variables
+real(dp),allocatable             :: zSoilReverseSign(:)        ! height at bottom of each soil layer, negative downwards (m)
+real(dp),dimension(12)           :: greenVegFrac_monthly       ! fraction of green vegetation in each month (0-1)
+real(dp),parameter               :: doubleMissing=-9999._dp    ! missing value
```

Comment:
We need to have `intergerMissing` and `realMissing` as global variables. This will be resolved as part of issue #126.